### PR TITLE
Saving DG values to file and plotting OpenFE against FEP+

### PR DESCRIPTION
**Pull request by hannahbaumann**
## Licensing agreement
* [x] I agree to release these inputs under the combined MIT and CCBY SA 4.0 licenses of this repository

[templates]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks
[input_validation]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks/utils/input_validation.py

### CODE REVIEW

File: industry_benchmarks/utils/extras/plot_results.py [hunk 1]
```diff
@@ -226,6 +226,88 @@ def plot_schrodinger_comparison(
...
+    """
+    graph = femap.to_legacy_graph()
+    writer = csv.writer(
+        filename,
+        delimiter="\t",
+        lineterminator="\n",
+    )
+    writer.writerow(["ligand", "DG (kcal/mol)", "uncertainty (kcal/mol)"])
+    # ToDo: Should the values be shifted?
+    for node in graph.nodes(data=True):
```
Comment (hannahbaumann) [inline industry_benchmarks/utils/extras/plot_results.py:251]: Here I wasn't sure whether to save the raw values, or the "shifted" DG values.

Comment (IAlibay) [reply industry_benchmarks/utils/extras/plot_results.py:251]: I would go with shifted since that better represents what we're plotting.

**Review by IAlibay — CHANGES_REQUESTED** [review]
Thanks this is good @hannahbaumann but I'm realising this might break the intended use case of some of our partners, i.e. they will want dG results _without_ passing in known schrodinger values.

I'm starting to wonder if it might be better to write out the dG results (unshifted) in the extract script?

**Comment by hannahbaumann** [comment]
ToDo: Add a test for extracting results and getting a ddG and a dG set of values.

**Comment by hannahbaumann** [comment]
The test is currently failing because the results 0, 1, 2 are just a copy of each other, leading to a standard deviation of zero and therefore a failure when doing the MLE. I have to replace the data by others, potentially @mikemhenry has some cleaned up results data of some system?

**Review by IAlibay — APPROVED** [review]

**Review by IAlibay — APPROVED** [review]
lgtm thanks!

**Pull request merged**